### PR TITLE
Port search component

### PR DESCRIPTION
**Pull request by HeyZoos**
Port over the search components 

Still to do
- proper styling
- choosing a mapping layer
- passing on time query args
- adding leaflet draw

### CODE REVIEW

**Comment by vforgione** [comment]
Please resolve those merge conflicts. I want to try this out.

**Comment by HeyZoos** [comment]
Would not recommend lol, but knock yourself out

File: assets/src/js/components/Explorer.vue [hunk 1]
```diff
@@ -140,7 +136,22 @@ export default {
     updateSearchResults: function (event) {
       this.searchResults = event;
     }
+  },
+
+  mounted: function () {
```
Comment (brlodi) [inline assets/src/js/components/Explorer.vue]: This makes me a sad panda.

Comment (HeyZoos) [reply assets/src/js/components/Explorer.vue]: My deepest apologies sad 🐼, I've extracted the map into its own component. I hope this improves things a bit.

**Pull request merged**